<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>137</width>
     <height>150</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="surnameLineEdit">
      <property name="placeholderText">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="placeholderText">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="patronymicLineEdit">
      <property name="placeholderText">
       <string>Отчество</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
